.github/workflows/build-and-push-fleetctl-docker.yml

```
@@ -40,5 +40,7 @@ jobs:
         run: make fleetctl-docker
 
       - name: Push to Docker
-        run: docker push fleetdm/fleetctl:${{ inputs.image_tag }}
+        run: |
+          docker tag fleetdm/fleetctl fleetdm/fleetctl:${{ inputs.image_tag }}
```

this is creating 2 tags, why?
```suggestion
          docker tag fleetdm/fleetctl:${{ inputs.image_tag }}
```

---

Thanks for clarifying. I misunderstood that docker command